setup.py

```diff
@@ -536,6 +536,8 @@ def run(self):
 
 setup_requires = ['setuptools', 'wheel']
 install_requires = [
+    "apispec; python_version >= '3'",
```

Comment:
Just thought that I'm not sure we need to pin this to Python 3? Only apispec v3 is Python 3 only

Reply:
Honestly, I'm not sure as I haven't tested on python 2.